voluptuous/error.py

```diff
@@ -21,14 +21,22 @@ class Invalid(Error):
 
     def __init__(self, message: str, path: typing.Optional[typing.List[str]] = None, error_message: typing.Optional[str] = None, error_type: typing.Optional[str] = None) -> None:
         Error.__init__(self, message)
-        self.path = path or []
-        self.error_message = error_message or message
+        self.path_value = path or []
+        self.error_message_value = error_message or message
```

Comment:
I think it would be preferable to make these private, `_path_value` and `_error_message_value`?

Reply:
Agree.